docs/content/latest/reference/configuration/yb-master.md

```diff
@@ -341,6 +335,36 @@ The number of replicas, or copies of data, to store for each tablet in the unive
 
 Default: `3`
 
+##### --yb_num_shards_per_tserver
+
+The number of shards per YB-TServer for each YCQL table when a user table is created.
+
+Default: `-1` (server internally sets default value). For servers with two or less CPU cores, then the default value is `4`. For four or more CPU cores, the default value is `8`. Local cluster installations, created using `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this option.
+
+{{< note title="Important" >}}
+
+This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+
+{{< /note >}}
+
+{{< note title="Note" >}}
+
+On a per-table basis, the [`CREATE TABLE ... WITH TABLETS = <num>`](../../../api/ycql/ddl_create_table/#with-tablets) clause can be used to override the `yb_num_shards_per_tserver` value.
```

Comment:
Is the link correct? Maybe it should be:
`../../../api/ycql/ddl_create_table/#create-a-table-specifying-the-number-of-tablets`

Reply:
With all of the changes and cross-references, it looks like you're right. Fixed.